django/chat/static/chat/style.css

```diff
@@ -1271,3 +1271,28 @@ math[display="block"] {
   margin-top: 1rem;
   padding-bottom: 1rem;
 }
+
+.wand-button {
+  background-color: transparent;
+  border: none;
+  top: 0px;
+  right: 40px;
+  width: 35px;
+  height: 30px;
+  display: flex;
+  position: absolute;
+  z-index: 5">
```

Comment:
Formatting error here probably from copy/paste. This is likely why the z-index wasn't working here.